src/main/index.ts

```diff
@@ -161,6 +166,28 @@ function postLoadConfig(): void {
   });
 }
 
+/**
+ * Save all the messages from the console log
+ * to a .log file
```

Comment:
lets fit this description into one line, it looks like it can fit within 100 characters